mlibrary/my_account
# Webpack JS Changes

# Overview
While adding more JavaScript to `./js/index.js`, I've noticed that functions used on other pages were throwing errors. I decided to create separate JavaScript files for what is used everywhere, and what is used only on certain pages, and then load those specific files on the specific pages.

If you can think of a better method for this, please let me know.

## Testing
- Run `npm run build-js`.
- Go to http://localhost:4567/current-checkouts/checkouts, and inspect to see if `<script src="/bundles/currentCheckouts.bundle.js"></script>` appears.
- Go elsewhere and inspect to see if `<script src="/bundles/currentCheckouts.bundle.js"></script>` does not show up.
- Double check to see if any console errors appear.
